azure-pipelines.yml

```diff
@@ -73,16 +73,16 @@ stages:
         parameters: # see template file for a definition of the parameters.
           stage_name: ci_build
           test_pool_definition:
-            vmImage: 'ubuntu-20.04'
+            vmImage: 'ubuntu-22.04'
```

Comment:
I wonder why we are not going to 24.04 -since it is the latest LTS release?

Reply:
because we already did the work required for Ubuntu 22.